contentcuration/contentcuration/frontend/shared/app.js

```diff
@@ -128,6 +128,9 @@ export default async function startApp({ store, router, index }) {
   }
 
   window.addEventListener('beforeunload', e => {
+    if (e.origin !== window.location.origin) {
```

Comment:
Is there actually an `origin` property on a `BeforeUnloadEvent` (`Event`)?

Reply:
Hrm, according to MDN, it seems not - Sonar Lint may have misled me here.

Reply:
Seems like `e.currentTarget.location.origin` should work!
![Screenshot from 2021-08-04 16-25-07](https://user-images.githubusercontent.com/819838/128268129-40911d16-c33d-4d44-921c-c4dec99e1d00.png)

Reply:
Only on MessageEvent: https://developer.mozilla.org/en-US/docs/Web/API/MessageEvent/origin

Reply:
Oh, let's do that then!

Reply:
```suggestion
    if (e.currentTarget.location.origin !== window.location.origin) {
```